pytools/icoscp/sparql/sparqls.py

```diff
@@ -724,3 +724,39 @@ def icos_hist_sparql():
     
     return query
 #------------------------------------------------------------------------------
+
+
+
+def get_icos_citation(dataObject):
+    
+    """
+    Project:         'ICOS Carbon Portal'
+    Created:          Fri May 10 12:35:00 2019
+    Last Changed:     Fri May 15 09:55:00 2020
+    Version:          1.1.0
+    Author(s):        Oleg, Karolina
+    
+    Description:      Function that takes a string variable representing the URL with data object ID as input and
+                      returns a query-string with the citation for the corresponding data object. The data object ID is
+                      a unique identifier of every separate ICOS dataset.
+    
+    Input parameters: Data Object ID (var_name: "dataObject", var_type: String)
+    
+    Output:           Citation (var_type: String)
+
+    """
+    
+    #Define URL:
+    url = 'https://meta.icos-cp.eu/sparql'
+    
```

Comment:
# check input variable 
dataObject = __dobjUrl__(dataObject)

Reply:
and again the double underscore has been removed from github.

Reply:
ok, to check if input variable is PID or url 
BUT in this case '<' and '>' need to be removed when composing the query string.